upgrading/osd-upgrades.adoc

```diff
@@ -9,6 +9,8 @@ toc::[]
 [role="_abstract"]
 You can schedule automatic or manual upgrade policies to update the version of your {product-title} clusters. Upgrading {product-title} clusters can be done through {cluster-manager-first} or {cluster-manager} CLI.
 
+The SRE team monitors upgrade progress and remedies any issues encountered.
```

Comment:
I am not very familiar with the mention of `the SRE team`, but I did a search in the docs and saw `Red Hat SRE teams` https://docs.openshift.com/container-platform/4.11/updating/understanding-upgrade-channels-release.html#stable-version-channel_understanding-upgrade-channels-releases 

I am not confident enough to say that is what should be used, but I thought I would call it out in case.

Reply:
I see further down in that link you provided that it reads `Red Hat Site Reliability Engineers (SRE)' that should probably be used for this one as well. https://51788--docspreview.netlify.app/openshift-dedicated/latest/upgrading/osd-upgrades.html#upgrade_osd-upgrades